Source/WebCore/style/StyleBuilderConverter.h

```diff
@@ -1840,7 +1840,9 @@ inline OptionSet<HangingPunctuation> BuilderConverter::convertHangingPunctuation
 
 inline GapLength BuilderConverter::convertGapLength(BuilderState& builderState, const CSSValue& value)
 {
-    return (value.valueID() == CSSValueNormal) ? GapLength() : GapLength(convertLength(builderState, value));
+    if (!value.isPrimitiveValue())
+        return { };
```

Comment:
Apparently, this isn't the right fix. We need to throw a `TypeError` in `attributeStyleMap.set`.